action.yml

```diff
@@ -1,38 +1,42 @@
 name: 'Add Navbar Action'
-description: 'Adds a top navigation bar to Documenter.jl-generated sites in the gh-pages branch.'
+description: 'Adds a top navigation bar to Documenter.jl-generated sites'
 inputs:
+  doc-path:
+    description: 'Path to the Documenter.jl output'
+    required: false
+    # Most julia projects have it here
+    default: 'docs/build'
   navbar-url:
     description: 'URL of the navbar HTML to be inserted.'
-    required: true
+    required: false
+    default: 'https://raw.githubusercontent.com/TuringLang/turinglang.github.io/main/assets/scripts/TuringNavbar.html'
```

Comment:
Because this action is only really meant to be used on TuringLang repos (IMO, this repo should eventually go onto the TuringLang org), I think we can afford to be opinionated about our default values. That will mean we don't have to copy-paste this link across every repo in TuringLang :)

Reply:
> Because this action is only really meant to be used on TuringLang repos (IMO, this repo should eventually go onto the TuringLang org), I think we can afford to be opinionated about our default values. That will mean we don't have to copy-paste this link across every repo in TuringLang :)

I agree with this! But, I was thinking about expanding this action to support more sites, like `readthedocs`, and provide different styles of navigation bars, as well as options for customizing the construction of the navigation bar. But that would be a longer-term goal, so let's keep Turing as the default for now!